scripts/extract-elements.js

```diff
@@ -25,9 +25,9 @@ function extractElements($) {
         'portal'
     ];
     // `<section>` is for some reason missing from the reference tables.
-    // `<command>` and `element` are obsolete and has been removed from the
-    // reference table, but we had them in the past so we should wait for a
-    // major to remove
+    // `<command>`, `<element>`, `<isindex>`, `<listing>`, `<multicol>`, `<nextid>`
+    // are obsolete and has been removed from the reference table, but we had
+    // them in the past so we should wait for a major to remove
```

Comment:
We do have a major coming up 😉 @jonmmease @HammadTheOne @chriddyp

Reply:
small nitpick, but "has been removed" should be "have been removed", :)

Reply:
thx -> https://github.com/plotly/dash-html-components/pull/194/commits/0ce630da4ab4e075259633c0a641a5d92c7d698a